src/components/pages/Dashboard/Analyze.tsx

```diff
@@ -11,20 +11,18 @@ import Charts from "../../charts/Charts";
 export default function Analyze() {
   const isMobileDeviceOrTablet = useMediaQuery({ maxDeviceWidth: mobileDeviceOrTabletWidth });
   const [state, dispatch] = useContext(AppContext);
-  
+
   useEffect(() => {
-    if(state.filtered_records.length > 0){
-      const updateCountryPrevalence = async () => {
-        const api = new httpClient();
-        const estimateGradePrevalences = await api.getEstimateGrades(state.filtered_records);
-        dispatch({
-          type: 'UPDATE_ESTIMATE_PREVALENCES',
-          payload: estimateGradePrevalences
-        });
-      } 
-      updateCountryPrevalence();
+    const updateCountryPrevalence = async () => {
```

Comment:
Figure out if we still need this lol, or rather, if it is best placed in analyze.tsx